migrations/20221121110412_modify_users_email_unique_index.up.sql

```diff
@@ -0,0 +1,16 @@
+-- this change is relatively temporary
+-- it is meant to keep database consistency guarantees until there is proper
+-- introduction of account linking / merging / delinking APIs, at which point
+-- rows in the users table will allow duplicates but with programmatic control
+
+alter table only {{ index .Options "Namespace" }}.users
```

Comment:
just wondering, what does `only` do here?

Reply:
> The name (optionally schema-qualified) of an existing table to alter. If ONLY is specified before the table name, only that table is altered. If ONLY is not specified, the table and all its descendant tables (if any) are altered. Optionally, * can be specified after the table name to explicitly indicate that descendant tables are included.

I believe it won't affect inherited tables (not that we're using them) -- but better to be explicit than implicit I guess.